llvm/lib/Target/AArch64/AArch64.td

```diff
@@ -85,6 +85,10 @@ def SMEUnsupported : AArch64Unsupported {
                       SME2Unsupported.F);
 }
 
+def MTEUnsupported : AArch64Unsupported {
+  let F = [HasMTE];
+}
+
```

Comment:
I can't see that MTEUnsupported is referenced from elsewhere in this commit - superfluous? If so, remove. Thanks.

Reply:
It is referenced in AArch64SchedOryon.td on line 33. The Oryon CPU does not support MTE.

Reply:
Thanks for the FIXME prompt, I got it fixed and did some cleaning, Just uploaded the code.

Reply:
Thanks for removing those. Can't see anything else obviously wrong. LGTM, Approved.